masonry/src/widget/portal.rs

```diff
@@ -311,8 +336,14 @@ impl<W: Widget> Widget for Portal<W> {
             LifeCycle::WidgetAdded => {
                 ctx.register_as_portal();
             }
-            //TODO
-            //LifeCycle::RequestPanToChild(target_rect) => {}
+            LifeCycle::RequestPanToChild(target) => {
+                let portal_size = ctx.size();
+                let content_size = ctx.get_raw_ref(&mut self.child).ctx().layout_rect().size();
+
+                // FIXME - Update scrollbars
```

Comment:
We need to do this in this PR, or explain why it's hard. The behaviour without that is quite bad.

Reply:
Done.